CONTRIBUTING.md

```diff
@@ -38,20 +38,22 @@ Use IntelliJ IDEA Community Edition/Ultimate.
    * [PsiViewer IntelliJ plugin](https://plugins.jetbrains.com/plugin/227-psiviewer) can be helpful to see parsing result on the `machete` file
      when running IntelliJ instance with the Git Machete plugin loaded.
 
-6. Enable annotation processing (for Lombok):
+6. Set Project SDK to JDK 17: `Project Structure > Project`
```

Comment:
I am reverting this one btw - it is better to specify it here.

In my case - I opened the project on ubuntu (I have not opened it for a long time there) where the JDK version was set to 11. I was trying to run gradle task (buildPlugin) but it was failing due to compiled code version issues. I musted switch (or even add it before) JDK 17 in the project settings.

Reply:
Hmmm okay but was the problem that you were facing https://github.com/JetBrains/gradle-grammar-kit-plugin/issues/97 or something else? Coz once [the fix (JetBrains/gradle-grammar-kit-plugin#98)](https://github.com/JetBrains/gradle-grammar-kit-plugin/pull/98) is released, then probably Gradle toolchain should work as expected (and hence there won't be a need to set JDK explicitly)... unless you discovered otherwise now? 🤔

Reply:
```Error: Unable to initialize main class org.intellij.grammar.Main```

yes, I remember this one

Reply:
Okay... so assuming that the fix to gradle-grammar-kit-plugin plugin is released soon(ish), is the `6. Set Project SDK to JDK 17` point needed?

Reply:
ok, but I will not be surprised if this happens to be a problem for someone before the change is released 🙄